extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/write/HiveWriteHelper.scala

```diff
@@ -59,24 +59,15 @@ object HiveWriteHelper extends Logging {
     // We have to follow the Hive behavior here, to avoid troubles. For example, if we create
     // staging directory under the table director for Hive prior to 1.1, the staging directory will
     // be removed by Hive when Hive is trying to empty the table directory.
-    val hiveVersionsUsingOldExternalTempPath: Set[HiveVersion] = Set(v12, v13, v14, v1_0)
```

Comment:
SPARK-45328(4.0.0) removes Hive support prior to 2.0.0, and deletes those enumeration values, we use Kyuubi's SemanticVersion to compare the Hive's version instead, to avoid linkage issues.